cmd/spire-server/cli/run/run_posix_test.go

```diff
@@ -4,18 +4,168 @@
 package run
 
 import (
+	"bytes"
+	"fmt"
 	"os"
+	"syscall"
 	"testing"
+	"time"
 
+	commoncli "github.com/spiffe/spire/pkg/common/cli"
+	"github.com/spiffe/spire/pkg/common/fflag"
+	"github.com/spiffe/spire/pkg/common/log"
 	"github.com/spiffe/spire/pkg/server"
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
 )
 
 const (
-	configFile = "../../../../test/fixture/config/server_good_posix.conf"
+	configFile  = "../../../../test/fixture/config/server_good_posix.conf"
+	testTempDir = "/tmp/spire-server-test"
```

Comment:
This should be handled by calling t.TempDir().

Reply:
In this specific test i wasn't able to to use  t.TempDir() because I needed to interpolate this dir path into the loaded conf file ( server_run_posix.conf), but I see that in another test case we used env vars for making this interpolation (server_good_templated.conf), I'm going to take the same approach here 👍